manual/manual/cmds/spacetime.etex

```diff
@@ -0,0 +1,118 @@
+\chapter{Memory profiling with Spacetime}
+\pdfchapterfold{-9}{Memory profiling with Spacetime}
+%HEVEA\cutname{spacetime.html}
+
+\section{Overview}
+
+Spacetime is the name given to functionality within the OCaml compiler that
+provides for accurate profiling of the memory behaviour of a program.
+Using Spacetime it is possible to determine the source of memory leaks
+and excess memory allocation quickly and easily.  Excess allocation slows
+programs down both by imposing a higher load on the garbage collector and
+reducing the cache locality of the program's code.  Spacetime provides
+full backtraces for every allocation that occurred on the OCaml heap
+during the lifetime of the program including those in C stubs.
+
+Spacetime only analyses the memory behaviour of a program with respect to
+the OCaml heap allocators and garbage collector.  It does not analyse
+allocation on the C heap.  Spacetime does not affect the memory behaviour
+of a program being profiled with the exception of any change caused by the
+overhead of profiling (see section\ \ref{runtimeoverhead})---for example
+the program running slower might cause it to allocate less memory in total.
+
+Spacetime is currently only available for x86-64 targets and has only been
+tested on Unix-like systems.  It is expected that supported platforms will
```

Comment:
Actually I thought that Spacetime was working on Windows (without support for building call graph within C code).  Is it not the case at all, or is it just that testing was not as thorough as for Unix-like system?